### PR TITLE
TTAHUB-188: Changes for Sequelize Join Issue

### PR DESCRIPTION
**Pull request by AdamAdHocTeam**
## Description of change
Added the "separate: true" parameter to included models in the getActivityReportById function in the backend code

JIRA #: TTAHUB-188

## How to test
To start, we've created a complex activity report at https://tta-smarthub-dev.app.cloud.gov/activity-reports/10
This activity report is large enough to crash the dev instance simply by visiting the above URL. We've recorded some rough benchmarks for current performance.

Our testing strategy will be to recreate the report in the sandbox, perhaps in stages, and observe the behavior and responsiveness of the app in whatever monitoring tools are available in that environment (New Relic, Kibana, etc)

Once that's been done, we will move on to testing in dev and confirm performance is improved. Again, we will verify and record any findings via logging tools.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-188

### Every PR

- [x] Meets issue criteria
- [x] JIRA ticket status updated
- [x] Code is meaningfully tested

**N/A**
- [ ] ~Meets accessibility standards (WCAG 2.1 Levels A, AA)~
- [ ] ~API Documentation updated~
- [ ] ~Boundary diagram updated~
- [ ] ~Logical Data Model updated~
- [ ] ~[Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions~

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [ ] Update JIRA ticket status

### CODE REVIEW

**Review by SarahJaine — APPROVED** [review]
I think this is good. I'm seeing in the [docs for `findOne`](https://sequelize.org/v5/class/lib/model.js~Model.html#static-method-findOne) that options are the same as for `findAll`.  And it looks like you added the separate param to all `hasMany` associations in this query. Can you say a little more about how you plan to test. We usually have step-by-step instructions in that section. I realize we can't test against dev until after merge, but it'd be nice to document some more details there.

**Review by SarahJaine — CHANGES_REQUESTED** [review]
Wait! I just remembered we have a new relic instance for sandbox - so we can test this there. Let's do that first.

**Review by SarahJaine — APPROVED** [review]
TLDR: approved.... testing this in sandbox first would probably be better, but it will also cause delay because you'll have to get access and create a big report in that environment. So I'll leave it up to you two to decide how to proceed.
-
More details:
I just remembered that sandbox is a monitored environment, so we could test this there instead and not risk negative impacts to dev.  I pushed a commit to this branch that will cause your code to be deployed to sandbox.  sandbox is available at https://tta-smarthub-sandbox.app.cloud.gov/, but you'll need to go through the login steps and use smartsheet to request access to this space as well. And you'd have to create a new, big report to test out in that space. (sandbox has its own database.)

**Comment by SarahJaine** [comment]
There's some explanation of sandbox in our readme https://github.com/adhocteam/Head-Start-TTADP#continuous-deployment-cd

**Comment by thewatermethod** [comment]
Increased express.json size limit re docs https://expressjs.com/en/4x/api.html#express.json

**Review by dcloud — APPROVED** [review]
The testing we've done in staging makes me feel confident we can take these changes to production. Good catch on the body request size issue, 2MB seems like plenty!

I would still like for us to measure the application performance when querying the db for large reports. I think we've mitigated the issue but could identify further improvements to be made.
